src/aliceVision/sfm/utils/alignment.cpp

```diff
@@ -1026,6 +1026,62 @@ void computeCentersVarCov(const sfmData::SfMData& sfmData, const Vec3 & mean, Ei
     }
 }
 
+void computeNewCoordinateSystemGroundAuto(const sfmData::SfMData& sfmData, Vec3& out_t)
```

Comment:
wouldn't be better to have the point returned instead as an out param?
```suggestion
Vec3 computeNewCoordinateSystemGroundAuto(const sfmData::SfMData& sfmData)
```